README.md

```diff
@@ -61,10 +61,49 @@ note is that the endpoint for submitting spot requests requires a fully formed
 and valid `LaunchSpecification`.
 
 ## Hacking
+
 ```
 git clone https://github.com/taskcluster/ec2-manager
 cd ec2-manager
-yarn
+yarn install --frozen-lockfile --ignore-optional
```

Comment:
my understanding is that `--ignore-optional` is something related a specific OS X environment?  My understanding is that doing so would cause `libpg` and `pg-native` to be ignored.  Since that's what we're using in production, I'd really like to have it installed during development.  Maybe we could add a note to say to use it the flag if you have install troubles, but calling out the issue around missing `pg-native`?